### DISCUSSION TITLE
oboeConfig.cmake not found

### DISCUSSION THREAD

**Discussion by Ateros4me** [google/oboe#1439] [Q&A]
When I tried to use the oboe library in my project the oboe directory appeared under the app section in Android Studio.
However, when I tried to compile the project including oboe, it failed with the following error  Could not find a package configuration file provided by "oboe" with any of
  the following names:

    oboeConfig.cmake
    oboe-config.cmake

  Add the installation prefix of "oboe" to CMAKE_PREFIX_PATH or set
  "oboe_DIR" to a directory containing one of the above files.  If "oboe"
  provides a separate development package or SDK, be sure it has been
  installed.

However, if I do a find in linux it finds oboeConfig.cmake in a path like this

${PROJECT_PATH}/app/.cxx/Debug/286r3a5v/prefab/x86/prefab/lib/i686-linux-android/cmake/oboe/oboeConfig.cmake
and the oboe-config.cmake isn't found at all.

This is the CMakeLists.txt file that I'm calling

[CMakeLists.txt](https://github.com/google/oboe/files/7498349/CMakeLists.txt)

It looks like the following line is causing the issue

find_package(oboe REQUIRED CONFIG)

**Comment by dturner** [comment]
It looks like you're confusing the [2 different methods of adding Oboe as a dependency](https://github.com/google/oboe/blob/main/docs/GettingStarted.md#adding-oboe-to-your-project). The `find_package` command will only succeed if you have added the prefab binaries in the dependencies section of your `build.gradle`

I would stick to the prefab method unless you're modifying the oboe source.
